src/components/Buttons/Button.tsx

```diff
@@ -43,6 +45,7 @@ export const Button: React.FC<ButtonProps> = ({
       disabled={disabled}
       onClick={onClick}
       id={id}
+      aria-describedby={ariaDescribedBy}
```

Comment:
<img width="1840" alt="Skærmbillede 2024-12-03 kl  09 32 25" src="https://github.com/user-attachments/assets/4890f49d-6019-4506-be4d-b13e5d775f82">

If it is 'undefined', it will not show 'aria-described by' at all. Therefore, it is important that we do not send an empty string, as the attribute would then be there without a value.

The aria-describedby attribute is typically used to provide additional context or description for a control beyond its label. For example: `findOnShelfExpandButtonExplanationText`